src/containers/Header/LanguagePicker/LanguagePicker.tsx

```diff
@@ -5,8 +5,16 @@ import './LanguagePicker.scss'
 import { supportedLanguages } from '../../../i18n/baseConfig'
 
 export const LanguagePicker = () => {
-  const { t, i18n } = useTranslation()
-  const currentLanguage = useLanguage()
+  const { i18n } = useTranslation()
+  const currentLanguage = useLanguage();
+  let lang:String;
+  if(currentLanguage=='en-US'){
+    lang  = "English";
```

Comment:
I would prefer this use a map that could be defined here https://github.com/ripple/explorer/blob/staging/src/i18n/baseConfig.ts#L3.

A map will eliminate the need for all these if statements.

Reply:
got it what you saying , lemme try again , thanks for feedback